fiftyone/utils/sam2.py

```
@@ -32,89 +32,76 @@
-        points_mask_index (None): an optional mask index to use for each
-            keypoint output
     """
 
-    def __init__(self, d):
-        d = self.init(d)
-        super().__init__(d)
-
-        self.auto_kwargs = self.parse_dict(d, "auto_kwargs", default=None)
-        self.points_mask_index = self.parse_int(
-            d, "points_mask_index", default=None
+    pass
```

Ruff (0.15.5)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)